bin/laminas

```diff
@@ -11,8 +11,12 @@ declare(strict_types=1);
 
 use Laminas\Cli\ContainerCommandLoader;
 use Laminas\Cli\ContainerResolver;
+use Laminas\Cli\LazyLoadingCommand;
 use PackageVersions\Versions;
 use Symfony\Component\Console\Application;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleTerminateEvent;
+use Symfony\Component\EventDispatcher\EventDispatcher;
```

Comment:
HA! I suggested yesterday do a PSR-14 dispatcher, but forgot that symfony/event-dispatcher provides an implementation as of v5, and that _it's already installed with symfony/console_! Nice!

Reply:
HA! It is PSR-14 but `setDispatcher` in `Symfony\Console\Application` does not typehint on PSR-14 event dispatcher but symfony contract event dispatcher...